File: lib/app/Models/UpdateOfferContentResponse.php
<?php
/**
 * UpdateOfferContentResponse
 */
namespace app\Models;

/**
 * UpdateOfferContentResponse
 * @description Описывает проблемы, которые появились при сохранении товара.
 */
class UpdateOfferContentResponse {

    /** @var string $status */
    public $status = "";

    /** @var \app\Models\UpdateOfferContentResultDTO[]|null $results Ошибки и предупреждения, которые появились при обработке переданных значений. Каждый элемент списка соответствует одному товару.  Если ошибок и предупреждений нет, поле не передается.*/
    public $results = null;

}

File: src/main/kotlin/ympa_kotlin_wiremock_server/models/UpdateOfferContentResponse.kt
@file:Suppress(
    "RemoveRedundantQualifierName",
    "unused",
)

package ympa_kotlin_wiremock_server.models

import com.fasterxml.jackson.annotation.JsonProperty

data class UpdateOfferContentResponse(
    @field:JsonProperty("status")
    val status: ApiResponseStatusType? = null,

    @field:JsonProperty("results")
    val results: kotlin.collections.List<UpdateOfferContentResultDTO>? = null,

)
